### PR TITLE
Update manifest.md

### PR DESCRIPTION
**Pull request by isamu**
## Description

I added the missing title to the documentation for the preference properties. I had set the preference.name according to the document, but the name was not displayed in the Raycast settings screen. After adding the title, it was displayed.

## Screencast

First line sets both name and title. Second line sets only the name.

スクリーンショット 2024-08-23 9 13 45

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: docs/information/manifest.md [hunk 1]
```diff
@@ -70,6 +70,7 @@ All properties for extension or command-specific preferences. Use the [Preferenc
...
 | --------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | name<mark style="color:red;">\*</mark>        | A unique id for the preference.                                                                                                                                                                                            |
+| title<mark style="color:red;">\*</mark>       | The display name of the preference, shown to the user in Raycast.                                                                                                                                                                                                                                                                                                                                                  |
```
Comment (jfkisafk) [inline docs/information/manifest.md:73]: I think we should move the title from `checkbox` additional property to here:

https://github.com/raycast/extensions/blob/9f4cf2803678ea15b84e4e13b31bfa454c32a843/docs/information/manifest.md?plain=1#L85

What do you think?

Comment (isamu) [reply docs/information/manifest.md:73]: I didn’t notice it was there. I agree it should be moved. I will copy the content from the checkbox section here and then delete the checkbox section. Is that okay?

Comment (jfkisafk) [reply docs/information/manifest.md:73]: Yes that's what I was thinking too.

Comment (isamu) [reply docs/information/manifest.md]: Thank, I updated it.

**Review by jfkisafk — APPROVED** [review]

File: docs/information/manifest.md (hunk 1 above)
Comment (jfkisafk) [reply docs/information/manifest.md]: For sure, you can resolve this comment now. I approved the changes.

Comment (isamu) [reply docs/information/manifest.md]: Thank you for your support!!

**Review by nagauta — APPROVED** [review]
thanks!

**Comment by pernielsentikaer** [comment]
Thanks for the correction

**Pull request merged**